ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html

```diff
@@ -1,91 +1,194 @@
+
+            <input id="docker-volname-selector"
+                   class="form-control"
+                   type="text"
+                   placeholder="Volume name"
+                   formControlName="dockerVolName">
+
+          </label>
+
+        </div>
+        <div class="col-xs-1 px-0 vol-datastore-actions">
+
+          <clr-icon class="is-solid"
+                    shape="times-circle"
+                    (click)="removeVolumeDatastoreEntry(i)"
+                    *ngIf="i > 0">
+          </clr-icon>
+          <clr-icon class="is-solid"
+                    shape="plus-circle"
+                    (click)="addNewVolumeDatastoreEntry()"
+                    *ngIf="i === form.controls.volumeStores.controls.length - 1">
+          </clr-icon>
+
         </div>
       </div>
-      <div class="fg-group">
-        <label class="p1">Base image size</label>
-        <div class="form-group row">
-          <div class="col-xs-3">
-            <label class="p5" for="base-image-size" aria-haspopup="true" role="tooltip" class="tooltip tooltip-validation tooltip-md tooltip-top-right"
-              [class.invalid]="form.get('baseImageSize').invalid && (form.get('baseImageSize').dirty || form.get('baseImageSize').touched)">
-              <input type="text" id="base-image-size" formControlName="baseImageSize" class="form-control">
-              <span class="tooltip-content">{{ formErrMessage }}</span>
-            </label>
-          </div>
-          <div class="col-xs-2">
-            <div class="select">
-              <select formControlName="baseImageSizeUnit">
-                <option value="MB">MB</option>
-                <option selected value="GB">GB</option>
-              </select>
-            </div>
-          </div>
+    </div>
+
+
+    <div class="form-group row">
+      <div class="col-xs">
+
+        <label>
+          Base image size
+        </label>
+
+      </div>
+      <div class="col-xs-3">
+
+        <label for="base-image-size"
+               aria-haspopup="true"
+               role="tooltip"
+               class="form-control tooltip tooltip-validation tooltip-md tooltip-top-right"
+               [class.invalid]="form.get('baseImageSize').invalid && (form.get('baseImageSize').dirty || form.get('baseImageSize').touched)">
+
+          <input type="text"
+                 id="base-image-size"
+                 formControlName="baseImageSize"
+                 class="form-control">
+          <span class="tooltip-content">{{ formErrMessage }}</span>
+
+        </label>
+
+      </div>
+      <div class="col-xs-2">
+
+        <div class="select form-control">
+          <select formControlName="baseImageSizeUnit">
+            <option value="MB">MB</option>
+            <option selected value="GB">GB</option>
+          </select>
         </div>
+
       </div>
+      <div class="col-xs-2"></div>
+      <div class="col-xs"></div>
```

Comment:
This is to set space for previous auto layout columns, so form controls align to the same column. I will add a markup comment per occurrence to explain this.